tests/config_files_examples/docs-basics.config.json

```
@@ -1,17 +1,20 @@
 {
   "index_uid": "docs",
-  "start_urls": ["https://docs.meilisearch.com"],
+  "start_urls": ["https://www.meilisearch.com/docs"],
```

```suggestion
  "start_urls": ["https://meilisearch.com/docs"],
```

Not sure this is required